src/mixins/filing-template-mixin.ts

```diff
@@ -251,7 +252,9 @@ export default class FilingTemplateMixin extends Vue {
       certifiedBy: filing.header.certifiedBy
     })
 
-    this.setDetailComment(filing.correction.comment)
+    const comment: string = filing.correction.comment || ''
+    const detailComment = comment.split('\n').slice(1).join('\n')
```

Comment:
This is complex enough that I think it needs a comment.
Might as well add a comment to line 110 to say what you're doing there too :)

Reply:
Absolutely, will do!